src/events/BasicMessageEvents.ts

```diff
@@ -11,7 +11,9 @@ export const basicMessageEvents = async (agent: Agent, config: ServerConfig) =>
 
     // Only send webhook if webhook url is configured
     if (config.webhookUrl) {
-      await sendWebhookEvent(config.webhookUrl + '/basic-messages', body, agent.config.logger)
+      for (const webhookUrl of config.webhookUrl) {
+        sendWebhookEvent(webhookUrl + '/basic-messages', body, agent.config.logger)
```

Comment:
so if there are multiple webhooks we no longer await the send webhook events?

Reply:
we never should have awaited it, since we don't want or need to wait for the response.